File: Facade/CustomerRelationship/UI/CustomerRelationship.Web/ClientApp/src/app/model/billing.ts
import { Customer } from "./Customer";

export class Billing {
  public id: number;
  public invoiceNumber: string;
  public emissionDate: Date;
  public dueDate: Date;
  public paymentDate?: Date;
  public description: string;
  public value: number;
  public receiptValue?: number;
  public increasedValue?: number;
  public discountValue?: number;

  public customerId: number;
  public customer: Customer;
}

File: Facade/CustomerRelationship/Domain/CustomerRelationship.Model/Entities/Billing.cs
using CustomerRelationship.Infrastructure.Model.BaseModel;
using System;

namespace CustomerRelationship.Model.Entities
{
    public class Billing : IdentificationBaseEntity
    {
        public string InvoiceNumber { get; set; }
        public DateTime EmissionDate { get; set; }
        public DateTime DueDate { get; set; }
        public DateTime? PaymentDate { get; set; }
        public string Description { get; set; }
        public decimal Value { get; set; }
        public decimal? ReceiptValue { get; set; }
        public decimal? IncreasedValue { get; set; }
        public decimal? DiscountValue { get; set; }

        public int CustomerId { get; set; }
        public virtual Customer Customer { get; set; }
    }
}
